### PR TITLE
Feature toggles: remove lokiMetricDataplane

### LINKED ISSUE #67211: Feature toggle removal: lokiMetricDataplane
enabled by default in https://github.com/grafana/grafana/pull/67137 on 2023-04-25, we should make a plan for removing it at the right time.

### PR DESCRIPTION
**Pull request by matyax**
On by default since 2023-04-25.

Closes https://github.com/grafana/grafana/issues/67211

### CODE REVIEW

**Review by svennergr — APPROVED** [review]
LGTM

File: pkg/tsdb/loki/framing_test.go [hunk 1]
```diff
@@ -72,10 +76,8 @@ func TestSuccessResponse(t *testing.T) {
 
 	for _, test := range tt {
 		t.Run(test.name, func(t *testing.T) {
-			runTest("testdata", test.filepath, test.query, ResponseOpts{metricDataplane: false, logsDataplane: false})
-			runTest("testdata_metric_dataplane", test.filepath, test.query, ResponseOpts{metricDataplane: true, logsDataplane: false})
-			runTest("testdata_logs_dataplane", test.filepath, test.query, ResponseOpts{metricDataplane: false, logsDataplane: true})
-			runTest("testdata_dataplane", test.filepath, test.query, ResponseOpts{metricDataplane: true, logsDataplane: true})
+			runTest("testdata_metric_dataplane", test.filepath, test.query, ResponseOpts{logsDataplane: false})
+			runTest("testdata_dataplane", test.filepath, test.query, ResponseOpts{logsDataplane: true})
```
Comment (svennergr) [inline pkg/tsdb/loki/framing_test.go:80]: While we are at it, do we wanna rename these tests/directories to something more descriptive?

File: pkg/tsdb/loki/framing_test.go [hunk 2]
```diff
@@ -64,6 +65,9 @@ func TestSuccessResponse(t *testing.T) {
 		bytes, err := os.ReadFile(responseFileName)
 		require.NoError(t, err)
 
+		fmt.Println("--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------")
+		fmt.Println(goldenFileName, responseFileName)
+
```
[the remark this answers is not in the document]
Comment (matyax) [reply pkg/tsdb/loki/framing_test.go:70]: 😳

File: pkg/tsdb/loki/framing_test.go (hunk 1 above)
Comment (matyax) [reply pkg/tsdb/loki/framing_test.go:76]: Absolutely. Any suggestion?

Comment (matyax) [reply pkg/tsdb/loki/framing_test.go:76]: I got it. I'm new to this part of the codebase so I wasn't sure of the original intention.

**Pull request merged**